### ISSUE TITLE
Should we add a watch task to the linter?

### ISSUE THREAD

**Issue by ebiven** [chaddm/schemafy#8]
Seems like that might be a nice addition. Having a pane running your tests on file change is helpful, why wouldn't I also want linting done? Or do we assume this is being handled by the editor?

**Comment by chaddm** [comment]
Yes, I think that is a great idea.  Unfortunately, I've know too many devs that don't have a linter in place.
